website/server/controllers/api-v3/inbox.js

```diff
@@ -0,0 +1,30 @@
+import { authWithHeaders } from '../../middlewares/auth';
+import * as inboxLib from '../../libs/inbox';
+
+let api = {};
+
+/* NOTE most inbox routes are either in the user or members controller */
+
+/* NOTE this route has also an API v4 version */
+/**
+ * @api {get} /api/v3/inbox/messages Get inbox messages for a user
+ * @apiName GetInboxMessages
+ * @apiGroup Inbox
+ * @apiDescription Get inbox messages for a user
+ *
+ * @apiSuccess {Array} data An array of <a href='https://github.com/HabitRPG/habitica/blob/develop/website/server/models/group.js#L51' target='_blank'>chat messages</a>
+ */
+api.getInboxMessages = {
```

Comment:
why do we have a v3 and v4 version of this route?

Reply:
It's a leftover from when they were behaving differently but the v4 one can be removed as the code is now the same